config/schema/synonyms.yml

```diff
@@ -102,6 +102,7 @@
 # Unofficial or deprecated terms should expand to the preferred term
 - search: ancestral visa => ancestry visa
 - search: bedroom tax => bedroom tax, spare bedroom, spare room
+- search: brexit => brexit, eu exit
```

Comment:
This is also configured in such a way that "EU Exit" is the preferred term for "Brexit". Is this the case given https://www.gov.uk/government/brexit ?
```suggestion
- search: eu exit => eu exit, brexit
```

Reply:
This block of configuration has the comment "Unofficial or deprecated terms should expand to the preferred term". My understanding looking at how other synonyms have been entered is that the unofficial or deprecated term is entered first.

So `search: brexit => brexit, eu exit` would suggest that `brexit` is the deprecated term, but any search for `brexit` should also include matches for `eu exit`.

You're right though, `brexit, eu exit => brexit, eu exit` might be what is needed since neither of these terms are used consistently. Or maybe it should be `eu exit, brexit => eu exit, brexit` ?!

Reply:
Yeah sorry this is a bit confusing. I put this synonym under this comment as Brexit is an unofficial term (like `xmas => xmas, christmas`). 

I changed it and moved it under the 'same meanings' comment.

Reply:
👍